target/postfix/main.cf

```diff
@@ -53,6 +53,8 @@ smtpd_client_restrictions = permit_mynetworks, permit_sasl_authenticated, reject
 smtpd_sender_restrictions = permit_sasl_authenticated, permit_mynetworks, reject_unknown_sender_domain, reject_unknown_client_hostname
 disable_vrfy_command = yes
 
+mua_sender_restrictions = reject_authenticated_sender_login_mismatch, $smtpd_sender_restrictions
```

Comment:
Wild guess that it Postfix doesn't like `smtpd_sender_restrictions` referencing itself :man_shrugging: 

```suggestion
dms_smtpd_sender_restrictions = permit_sasl_authenticated, permit_mynetworks, reject_unknown_sender_domain, reject_unknown_client_hostname
mua_sender_restrictions = reject_authenticated_sender_login_mismatch, $dms_smtpd_sender_restrictions
```